cves/CVE-2019-11339.yml

```diff
@@ -22,7 +22,7 @@ curated_instructions: |
 
   The latest curation level is 1.0.
   If you are curating this vulnerability, set it to the latest curation level.
-curation_level: 0.0
+curation_level: 1.0
```

Comment:
I'm going to stop here - I think this needs a lot more depth in it. Spend more time observing and researching the subsystem. Put yourself into a developer who is fixing their code they wrote a long time ago and trying to learn from it.

Reply:
I saw this when you posted it, so I went through and tried to add a more in-depth look at both of the CVE's I was working on.